Taskfile.yaml

```
@@ -109,7 +111,7 @@ tasks:
     internal: true
 
   schema:
-    deps: [schema-check]
+    deps: [ schema-check ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)